amm/src/main/scala/ammonite/interp/Preprocessor.scala

```diff
@@ -289,16 +304,16 @@ object Preprocessor{
                printCode: String,
                imports: Imports) = {
 
-    val topWrapper = s"""
+    val topWrapper = normalizeNewlines(s"""
 package ${pkgName.map(_.backticked).mkString(".")}
 ${importBlock(imports)}
 
-object ${indexedWrapperName.backticked}{\n"""
+object ${indexedWrapperName.backticked}{\n""")
 
-    val bottomWrapper = s"""\ndef $$main() = { $printCode }
+    val bottomWrapper = normalizeNewlines(s"""\ndef $$main() = { $printCode }
   override def toString = "${indexedWrapperName.raw}"
 }
-"""
+""")
```

Comment:
Need to normalize even as even if I remove `\n`s since this string is written on a linux machine the formatting due to `"""` will add `\n`s like 
`
"""
A

B
"""
`
will become "\nA\n\nB\n", so anyway normalize is necessary